tools/license-headers/Apache-2.0

```diff
@@ -0,0 +1,15 @@
+# sed -E -n
+# -e '/^Licensed under the Apache License/,/^limitations under the License./p'
```

Comment:
The Copyright line in the header is not omittable, only partly replaceable, so you should not skip it.

The anchoring of `limitations under the License.`to the beginning of line won't produce the results you expect.  You should either anchor this to the end of line (with `.` quoted), or search  for `Unless required by applicable` (this would be better, I think).  Please note the `license-detector` normalizes the text before it is handed over to the embedded processor.